sklearn/impute.py

```diff
@@ -38,12 +39,20 @@
 ]
 
 
+def _custom_isnan(x):
+    # np.nan is never equal to np.nan. Return true only if x is np.nan
+    return x != x
```

Comment:
You should use the one you imported from `.utils.fixes` instead of redefining this function here.

Reply:
I did not see the leading `_`. Anyway, this function seems to be unused and should, therefore, be deleted.